Twig/Extension.php

```diff
@@ -176,4 +183,82 @@ public function isRtl($lang = false): bool
         }
         return $this->intuition->isRtl($this->intuition->getLang());
     }
+
+    /**
+     * Get the currently checked-out Git branch.
+     * @return string
+     */
+    public function gitBranch(): string
+    {
+        $process = new Process('git rev-parse --symbolic-full-name --abbrev-ref HEAD');
+        $process->run();
+        return trim($process->getOutput());
+    }
+
+    /**
+     * Get the full hash of the currently checked-out Git commit.
+     * @return string
+     */
+    public function gitHash(): string
+    {
+        $process = new Process('git rev-parse HEAD');
+        $process->run();
+        return trim($process->getOutput());
+    }
+
+    /**
+     * Get the short hash of the currently checked-out Git commit.
+     * @return string
+     */
+    public function gitHashShort(): string
+    {
+        $process = new Process('git rev-parse --short HEAD');
```

Comment:
I noticed just recently that IntelliJ was complaining that `Process` is supposed to only accept an array. Not important, and really, I don't like the array format. Just letting you know. I'm content with ignoring it.